UnitTestBot/UTBotJava
# try to build ultimate plugin each time on pr event #1836

## Description

We need to check if IU plugin is built ok

Fixes #1836

## How to test

### Automated tests

Not applicable

### Manual tests

Check workflow from branch. Temporary change behavior of artifact publishing to be sure it works

## Self-check list

- [x] check tests from branch
- [x] test artifacts publishing from branch
- [x] disable artifact publishing from a branch before merge this pr
